pkg/reconciler/instances/istio/README.md

```diff
@@ -30,3 +30,33 @@ Follow these steps to run Istio Reconciler on your local machine:
    ```bash
     ./bin/mothership-darwin local --value global.ingress.domainName=example.com,global.domainName=example.com --components cluster-essentials,istio
    ```
+
+## Details
+
+Reconciliation in Kyma is handled by Reconciler. The Mothership Reconciler knows the reconciliation status of every managed Kyma cluster and initiates reconciliation of all Kyma components.
+
+The reconciliation of Istio is split into two different Kyma components: Istio and Istio Resources. The separation enables having the pure installation process of Istio under control via Istio Reconciler implementation and the Reconciler framework. The Reconciler framework functionality simply applies charts to the cluster for Istio Resources.
```

Comment:
```suggestion
The reconciliation of Istio is split into two different Kyma components: **Istio** and **Istio Resources**. With this separation, you control the pure installation of Istio with the Istio Reconciler implementation and the Reconciler framework. The Reconciler framework functionality simply applies charts to the cluster for Istio Resources.
```

Trying to simplify that middle sentence. It's still not perfectly clear to me though.

I'm struggling to distinguish when we mean "Istio" in a general sense, and "Istio" as one of the two Istio-related components. I'm trying to support this by emphasizing the components but not sure that's the best solution.